translations/pl.lang

```diff
@@ -88,46 +88,46 @@ lang_title=Język
 lang_desc=Dostępne języki:
 lang_thislang=Polski 
 lang_success=Pomyślnie zmieniono język!
-hours=hours
-minutes=minutes
-seconds=seconds
-rank_rank=Rank
-rank_level=Level
-needs_manage_roles=You need to give the bot the Manage Roles permission to use this feature!
-rank_roles_title=Rank Roles
-rank_roles_setup=Use the ``rankrole`` command to setup Rank Roles.
-rank_role_title=Rank Role
-rank_role_usage=rankrole <config>. If you want to clear your Rank Roles, you can set config to ``none``. Config is a set of rules for your rank roles. For example, ``playtime_20:12345678912345678, sodas_10:12345678912345679`` will give the ``20`` people with the highest ``playtime`` the role with the ID ``12345678912345678``, then the ``10`` people with the most ``sodas`` the role ``12345678912345679``.
-rank_role_invalid_rule=This contains an invalid rule. Rules should follow the following format: ``metric_number:roleID``.
-rank_role_invalid_metric=Your config contains an invalid ``metric``. The following are valid metrics: ``kills``, ``deaths``, ``sodas``, ``medkits``, ``adrenaline``, ``rounds``, ``playtime``, ``escapes``, ``leaderboard``, ``fastestescape``.
-rank_role_invalid_number=Your config contains an invalid ``number``. Your ``number`` should be greater than zero but less than 100.
-rank_role_invalid_role=Your config contains an invalid role ID!
-rank_role_success=Successfully updated rank roles!
-warnings_title=Warnings
-warnings_footer=Warnings for 
-warnings_message_top=ID | Type | Message | Ban Length
-warnings_type_warning=Warning
+hours=godziny
+minutes=minuty
+seconds=sekundy
+rank_rank=Miejsce
+rank_level=Poziom
+needs_manage_roles=Musisz przydzielić botowi Zarządzanie Rolami!
+rank_roles_title=Role za miejsce w rankingu
+rank_roles_setup=Użyj komendy ``rankrole`` by przydzielić role za miejsce w rankingu.
+rank_role_title=Rola za miejsce w rankingu
+rank_role_usage=rankrole <opcja>. Jeśli chcesz wyczyścić role za miejsce w rankingu, możesz użyć opcji ``none``. Komenda pozwala na ustawienie zasad, na podstawie których przyznawane są role za miejsce w rakingu. Na przykład, ``playtime_20:12345678912345678, sodas_10:12345678912345679`` przydzieli ``20`` graczom z najwyższym ``czasem gry`` rolę z ID ``12345678912345678``, a następnie ``10`` graczom z najwyższą liczbą wypitych ``SCP-207`` rolę z ID ``12345678912345679``.
+rank_role_invalid_rule=Błędna opcja. Obowiązuje następujący format: ``filtr_liczba:IDroli``.
+rank_role_invalid_metric=Konfiguracja posiada błędny ``filtr``. Dostępne filtry: ``kills``, ``deaths``, ``sodas``, ``medkits``, ``adrenaline``, ``rounds``, ``playtime``, ``escapes``, ``leaderboard``, ``fastestescape``.
+rank_role_invalid_number=Konfiguracja posiada błędną ``liczbę``. Wybrana ``liczba`` powinna być większa od 0 i mniejsza od 100.
```

Comment:
The text inside of \`\` \`\` should be left untranslated as it is meant to include information about what actually goes into the command arguments.